script/aws_sts.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
lets rename folder `script -> scripts` in the future we may have other scripts

Reply:
done